packages/storybook8/stories/Components/SendBox/Docs.mdx

```diff
@@ -0,0 +1,72 @@
+import { SendBox } from '@azure/communication-react';
+import { Title, Description, Props, Heading, Source, Canvas, Meta, ArgsTable } from '@storybook/addon-docs';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+import { DetailedBetaBanner } from '../../BetaBanners/DetailedBetaBanner';
+import { AttachmentUploadsExample } from './snippets/AttachmentUploads.snippet';
+import { CustomIconExample } from './snippets/CustomIcon.snippet';
```

Comment:
These snippet imports can be removed from here, as they are only used in index.stories.ts